administrator/components/com_modules/src/Controller/ModulesController.php

```diff
@@ -55,7 +55,7 @@ public function duplicate()
 			$this->app->enqueueMessage($e->getMessage(), 'warning');
 		}
 
-		$this->setRedirect('index.php?option=com_modules&view=modules');
+		$this->setRedirect('index.php?option=com_modules&view=modules'. $this->getRedirectToListAppend());
```

Comment:
```suggestion
		$this->setRedirect('index.php?option=com_modules&view=modules' . $this->getRedirectToListAppend());
```
This will fix PHP CS check in drone.

Reply:
Thanks!